packages/docusaurus/bin/docusaurus.mjs

```diff
@@ -244,7 +244,12 @@ if (!process.argv.slice(2).length) {
 cli.parse(process.argv);
 
 process.on('unhandledRejection', (err) => {
+  // TODO this logger thing is annoying
+  // This makes it impossible to use native logging of nested stacktraces
+  // ie the error causes are not logged here
+  // It seems we can't have colors + nested stacktrace logging at the same time
   logger.error(err instanceof Error ? err.stack : err);
```

Comment:
@Josh-Cena this thing is a bit annoying because if an error is thrown with a cause it won't print the cause

Reply:
I believe we can use the built-in error logging? This code has been there forever, and I think it's fine at this point to remove it and just use `console.log(err)` instead

Reply:
Let's do that then, doesn't look too bad:

<img width="979" alt="CleanShot 2023-04-07 at 18 42 43@2x" src="https://user-images.githubusercontent.com/749374/230645721-190d3f2a-d6fc-46d0-9298-d4a5290cc6f3.png">